src/test/java/me/karakelley/http/RequestTest.java

```diff
@@ -2,78 +2,113 @@
 
 import org.junit.jupiter.api.Test;
 
-import java.io.IOException;
 import java.util.HashMap;
+import java.util.Map;
+
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
 class RequestTest {
   Request request;
 
   @Test
   void testGetMethod() {
-    request = new Request(HttpMethod.GET, null, null, null, "".getBytes(), 0);
+    request = basicGetRequest();
```

Comment:
My first thought was to suggest all these tests could be deleted, since I thought they were just testing getters on constructor arguments. Then I realized that you're actually going through the `Builder`, so they make a little more sense, *but* I might just make a single test with several assertions. Something like `testBuilderSetsAppropriateRequestAttributes`. Just seems a little overkill... @aaronlahey do you agree or disagree?

Reply:
I think I've done it both ways in the past. I don't have a particularly strong opinion (shocker, I know).